plugins/vivisect-broker/broker/vivserver.proto

```diff
@@ -8,12 +8,18 @@ message RunInstructionsRequest {
     int32 numInstructions = 1;
 }
 
+message MemoryAccess {
+    uint64 vAddr = 1;
+    uint32 size = 2;
```

Comment:
Can this field be optional or does mcad absolutely require it to work?

Reply:
For context, it took me a while to realize if vivisect readily exposed this info so it may be something that a generic emulator might not provide in a useful form.

Reply:
I understand. For checking two overlapping accesses we currently perform something akin to -

`first.endAddr <= second.startAddr + second.size` ([Reference](https://github.com/securesystemslab/llvm-project/blob/dev-incremental-mca/llvm/include/llvm/MCA/HardwareUnits/LSUnit.h#L76))

I suppose in the case that we dont have this information, we might have to assume some sane defaults.